### PR TITLE
feat:  Get CRD both for cli and api for existing resources

### PR DESCRIPTION
**Pull request by vsukhin**
## Pull request description 

Get CRD both for cli and api for existing resources

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: internal/app/api/v1/executor.go [hunk 1]
```diff
@@ -113,19 +127,42 @@ func mapExecutorCRDToExecutorDetails(item executorv1.Executor) testkube.Executor
...
+		return s.Error(c, http.StatusBadRequest, err)
 	}
+
+	c.Context().SetContentType(mediaTypeYAML)
+	return c.SendString(data)
+}
+
+func (s TestkubeAPI) getExecutorCRDs(c *fiber.Ctx, executors []testkube.ExecutorCreateRequest) error {
+	data := ""
+	firstEntry := true
```
Comment (exu) [inline internal/app/api/v1/executor.go:146]: Looks very similar to `common.UIPrintCRD` logic

File: internal/app/api/v1/tests.go [hunk 2]
```diff
@@ -467,3 +481,43 @@ func GetSecretsStringData(content *testkube.TestContent) map[string]string {
...
+	if err != nil {
+		return s.Error(c, http.StatusBadRequest, err)
+	}
+
+	c.Context().SetContentType(mediaTypeYAML)
+	return c.SendString(data)
+}
+
+func (s TestkubeAPI) getTestCRDs(c *fiber.Ctx, tests []testkube.Test) error {
+	data := ""
```
Comment (exu) [inline internal/app/api/v1/tests.go:500]: UIPrintCRD?

File: internal/app/api/v1/testsuites.go [hunk 3]
```diff
@@ -854,3 +868,43 @@ func mapTestStepToCRD(step testkube.TestSuiteStep) (stepSpec testsuitesv1.TestSu
...
+	if err != nil {
+		return s.Error(c, http.StatusBadRequest, err)
+	}
+
+	c.Context().SetContentType(mediaTypeYAML)
+	return c.SendString(data)
+}
+
+func (s TestkubeAPI) getTestSuiteCRDs(c *fiber.Ctx, testSuites []testkube.TestSuite) error {
+	data := ""
```
Comment (exu) [inline internal/app/api/v1/testsuites.go:887]: UIPrintCRD

File: internal/app/api/v1/webhook.go [hunk 4]
```diff
@@ -95,3 +104,35 @@ func (s TestkubeAPI) DeleteWebhooksHandler() fiber.Handler {
...
+	if err != nil {
+		return s.Error(c, http.StatusBadRequest, err)
+	}
+
+	c.Context().SetContentType(mediaTypeYAML)
+	return c.SendString(data)
+}
+
+func (s TestkubeAPI) getWebhookCRDs(c *fiber.Ctx, webhooks []testkube.Webhook) error {
+	data := ""
```
Comment (exu) [inline internal/app/api/v1/webhook.go:119]: UIPrintCRD.

**Comment by exu** [comment]
Decreased coverage ;)

**Comment by exu** [comment]
@vsukhin some minor (possible) refactorings + coverage decrease

`get*****CRD` methods looks like could be part of `common.UIPRintCRD`

**Review by exu — APPROVED** [review]
Minor comments

File: internal/app/api/v1/executor.go (hunk 1 above)
Comment (vsukhin) [reply internal/app/api/v1/executor.go]: refactored

File: internal/app/api/v1/tests.go (hunk 2 above)
Comment (vsukhin) [reply internal/app/api/v1/tests.go]: refactored

File: internal/app/api/v1/testsuites.go (hunk 3 above)
Comment (vsukhin) [reply internal/app/api/v1/testsuites.go]: refactored

**Pull request merged**